### DISCUSSION TITLE
Скрыть иконки глаза и печать для пользователей

### DISCUSSION THREAD

**Discussion by erpic-ru** [totumonline/totum-ru-issues-and-discussions#296] [Как это сделать?] [answered]
Подскажите, плз, можно ли скрыть иконки глаза и печати для отдельных таблиц?
<img width="73" alt="Снимок экрана 2022-12-21 в 09 53 24" src="https://user-images.githubusercontent.com/91674114/208840364-f0ff0d30-9ca6-4365-95b0-e28485fe070c.png">

**Comment by ttm-alexeypolunin** [comment, marked as answer]
При вызове в iframe через `linkToTable` и `linkTodataTable` — параметр называется `topbuttons: false`

Но для `linkTodataTable` он не пашет — мы уже исправили и завтра загрузим.

 Просто отключить нельзя, но я думаю нам нужно добавить параметр в форматирование таблицы.

**Reply by erpic-ru** [reply]
В анонимной таблице было нужно, но уже не сильно

**Reply by ttm-alexeypolunin** [reply]
В 4.8.51 — `topbuttons:false` для `setTableFormat`

<img width="319" alt="Снимок экрана 2022-12-27 в 15 32 45" src="https://user-images.githubusercontent.com/55755565/209667261-0ca5ec84-ca02-46e8-8a17-5d1964c8da49.png">
